### PR TITLE
Allow build jobs to fail

### PR DESCRIPTION
**Pull request by ivan-c**
Allow build jobs to fail until I can figure the best way to clone to an exact commit

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -55,9 +55,9 @@ matrix:
       addons:
         postgresql: "9.4"
         sauce_connect: true
-  # Build artifacts only (no testing)
   allow_failures:
     - env: TOXENV=ui
+    - env: TOXENV=build
```
Comment (pbugni) [inline .travis.yml]: `build` is a little ambiguous - can we rename that target (line 50 iiuc) to say `docker_build`?

Comment (ivan-c) [reply .travis.yml]: what about `build_artifacts` - technically we're building a deb package *and* a docker image

Comment (pbugni) [reply .travis.yml]: great - i think that's more clear.

**Comment by pbugni** [comment]
sure - thanks.

**Pull request merged**